compose/config/config.py

```diff
@@ -1154,6 +1154,20 @@ def merge_deploy(base, override):
     return dict(md)
 
 
+def merge_networks(base, override):
+    merged_networks = {}
+    all_network_names = set(base) | set(override)
+    base = {k: {} for k in base} if isinstance(base, list) else base
+    override = {k: {} for k in override} if isinstance(override, list) else override
+    for network_name in all_network_names:
+        md = MergeDict(base.get(network_name, {}), override.get(network_name, {}))
+        md.merge_scalar('aliases')
```

Comment:
This needs to be `md.merge_field('aliases', merge_unique_items_lists, [])` so the result of `aliases; ['a', 'b']` and `aliases: ['b', 'c']` is `aliases: ['a', 'b', 'c']` to be consistent with other, similar config lists.

Reply:
Cool! 
Actually I understood that the field would be overwritten instead of merged in this case. My bad, I should have asked before the implementation and not assumed that :/